Mathlib/Algebra/Ring/Equiv.lean

```diff
@@ -78,7 +78,7 @@ add_decl_doc RingEquiv.toMulEquiv
 /-- `RingEquivClass F R S` states that `F` is a type of ring structure preserving equivalences.
 You should extend this class when you extend `RingEquiv`. -/
 class RingEquivClass (F R S : Type*) [Mul R] [Add R] [Mul S] [Add S] [EquivLike F R S]
```

Comment:
If we replace all the `_EquivClass`es with `_HomClass`es, what should `RingEquivClass` become? I don't believe there is already a class for `map_add`+`map_mul`.